README.md

```diff
@@ -47,94 +49,46 @@ The eFiling hub is a foundational component to enhance citizen experiences for t
 
 ## Running the App
 
-By default a demo mode is enabled.
+By default a demo mode is enabled. <!-- is this configured by MVN_PROFILE or DEMO_MODE?-->
 
-First create a local `.env` at the root of the repository based off `.env.template`.
+First create a local `.env` at the root of the repository based off [.env.template](.env.template).
 
 Configure Keycloak
 
 ```bash
 docker-compose up -d --build keycloak
 ```
 
-go to [Efiling-api credentials](http://localhost:8081/auth/admin/master/console/#/realms/SpringBootKeycloak/clients/b7fd5f2f-d047-4916-a35e-0f7c622dfb5d/credentials) and click generate Secrets
+go to [Efiling-api credentials](http://localhost:8081/auth/admin/master/console/#/realms/SpringBootKeycloak/clients/b7fd5f2f-d047-4916-a35e-0f7c622dfb5d/credentials) and sign in. The default credentials to a new keycloak instance are:
+
+    username: admin
+    password: admin
 
-Copy the value of the secret to the `KEYCLOAK_CREDENTIALS_SECRET` in your `.env` file
+After signing in, click Regenerate Secret. Copy the value of the secret to the `KEYCLOAK_CREDENTIALS_SECRET` in your `.env` file
 
 Create a user [here](http://localhost:8081/auth/admin/master/console/#/create/user/SpringBootKeycloak)
 
 Click on the Credentials tab
 
 Set the password and set Temporary OFF
 
-Click Reset Password
-
-Create your known hosts for sftp upload. In command prompt navigate to /users/<YOURNAME>/.ssh/ and run command:
-
-```
-ssh-keyscan -p 23 localhost  >> known_hosts
-```
-
-If you want to integrate with the CSO application change the `MVN_PROFILE` to `default`
+Click Reset Password <!-- Why do we need to do this? This just undoes the previous step right?-->
```

Comment:
This section talks about standing up keycloak locally, but we're currently using a Keycloak running in OpenShift. Do we need this section if we aren't currently using local Keycloak?

Reply:
We're planning on using local Keycloak eventually, leaving this for now.